.yamllint

```diff
@@ -12,6 +12,8 @@ ignore: |
   node_modules/
   test/**/states/**/*.sls
   .kitchen/
+  kitchen.yml
+  test/salt/pillar/default.sls
```

Comment:
I believe this is avoidable as well.  The proposal here would be to have a separate test pillar file for Windows, which is used from a second suite, which still uses the `default` suite as the verifier.  We've got this setup in a number of formulas, such as:

* `iscsi-formula`
* `locale-formula`
* `php-formula`
* `rkhunter-formula`
* `template-formula`

Reply:
I really hate unnecessary duplications and believe the combined pillar more accurately represents real-world Salt usage (at least for me, anyway).

Reply:
A couple of angles here:

1. [PillarStack](https://docs.saltstack.com/en/latest/ref/pillar/all/salt.pillar.stack.html) is common in real-world Salt usage; having multiple pillar files here is a similar approach.
1. We could solve the duplication by having a third pillar file with all of the common values; then use the common + specific pillar files depending on the platform.

Reply:
What is the concern here? - is it to maximise the number of files being linted, or is there some best practice that I'm not aware of for avoiding Jinja in pillar files?
Personally, I would prefer to leave it as it is.

Reply:
(Discussed in Slack, including the plans for `config.get`, etc.).  We can revisit this later, no need to hold up this PR any longer.